src/Sylius/Behat/Context/Api/Shop/HomepageContext.php

```diff
@@ -22,7 +22,7 @@
 final class HomepageContext implements Context
 {
     public function __construct(
-        private ApiClientInterface $client,
+        private ApiClientInterface $productsClient,
```

Comment:
IMO it should be named as it was:
```suggestion
        private ApiClientInterface $client,
```
because API clients in Behats have been refactored and now, on this branch, we have only two, for shop and admin instead of having client per API resource as it was before